### DISCUSSION TITLE
Why is latest.json not generated sometimes?

### DISCUSSION THREAD

**Discussion by HuakunShen** [tauri-apps/tauri#10804] [Q&A]
I am using the tauri action to build and generate prod build for a Tauri v2 app.
Sometimes the latest.json isn't generated, and sometimes it does, not sure what I did wrong or what I've changed to cause that.

When `latest.json` isn't generated, the `.sig` files are also not generated.

I do have `tauri-plugin-updater = "2.0.0-rc"` installed, and `createUpdaterArtifacts` enabled in `tauri.conf.json`
```json
"plugins": {
		"updater": {
			"createUpdaterArtifacts": true,
			"endpoints": [],
			"pubkey": ""
		}
	}
```

**Comment by FabianLars** [comment]
Do you have the logs of the "failed" runs? Maybe also use the `--verbose` flag for a while (until the next failed run). What i could imagine is that the action context of the runs is somehow different and maybe missing the signing env var 🤔

**Reply by HuakunShen** [reply]
I figured it out, it was my mistake.
The `createUpdaterArtifacts` is under updater, but it should be under bundle.
Thank you!
